docs/api.md

```diff
@@ -11,20 +11,25 @@ For information about the Antrea API versioning policy, please refer to this
 
 These are the API group versions which are curently available when using Antrea.
 
-| API group | API version | API Service? | Introduced in | Deprecated in / Planned Deprecation | Planned Removal |
-|---|---|---|---|---|---|
-| `clusterinformation.antrea.tanzu.vmware.com` | `v1beta1` | No | v0.3.0 | N/A | N/A |
-| `core.antrea.tanzu.vmware.com` | `v1alpha2` | No | v0.11.0 | N/A | N/A |
-| `controlplane.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.10.0 | v0.11.0 | Aug 2021 |
-| `controlplane.antrea.tanzu.vmware.com` | `v1beta2` | Yes | v0.11.0 | N/A | N/A |
-| `networking.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.3.0 | v0.10.0 | Jun 2021 |
-| `ops.antrea.tanzu.vmware.com` | `v1alpha1` | No | v0.8.0 | N/A | N/A |
-| `security.antrea.tanzu.vmware.com` | `v1alpha1` | No | v0.8.0 | N/A | N/A |
-| `stats.antrea.tanzu.vmware.com` | `v1alpha1` | Yes | v0.10.0 | N/A | N/A |
-| `system.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.5.0 | N/A | N/A |
+| API group                                    | API version | API Service? | Introduced in | Deprecated in / Planned Deprecation | Planned Removal |
+| -------------------------------------------- | ----------- | ------------ | ------------- | ----------------------------------- | --------------- |
+| `clusterinformation.antrea.tanzu.vmware.com` | `v1beta1`   | No           | v0.3.0        | N/A                                 | N/A             |
+| `core.antrea.tanzu.vmware.com`               | `v1alpha2`  | No           | v0.11.0       | N/A                                 | N/A             |
```

Comment:
aren't these deprecated as well?

Reply:
I missed `core.antrea.tanzu.vmware.com`, and it should be deprecated. For `clusterinformation.antrea.tanzu.vmware.com`, I didn't modify it. It should not be deprecated.

Reply:
Aren't the CRDs in `clusterinformation.antrea.tanzu.vmware.com` being moved to `crd.antrea.io`?

Reply:
`clusterinformation.antrea.tanzu.vmware.com` is not a CRD, and users don't use it, so it is not neccessary to change it.

Reply:
Done.